ListLinked/LinkedList.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+namespace LinkedList
+{
+    public  class LinkedList<T> : IEnumerable<T>
+    {
+        private int position = 0;
+        public Node<T> head;
+
+
+        public void Add(T value)
+        {
+            if (head == null)
+            {
+                head = new Node<T>(value);
+                return;
+            }
+            Node<T> current = head;
```

Comment:
These days on many projects people use var over the explicit type declaration as a convention. But you are the one who drives the conventions here so choose either way and make sure you use it everywhere throughout this solution.